server.js

```diff
@@ -102,6 +102,35 @@ server.route([
     }
   }, {
     method: 'POST',
+    path: '/api/stripe-checkout',
```

Comment:
Stripe checkout is never going to have address, so instead of making address bits optional in the stripe route, I created a new one that only has what we need.

So if we try to use stripe checkout WITH address, it'll fail.